### PR TITLE
Event Single Reskin DQA Feedback

### PR DESCRIPTION
**Pull request by EugeneKyale**
This PR addresses the following issues that came up during DQA. 

- Reduce the space between the venue details and map in desktop. 
- Updates the recurring event pill to the updated design.

Ticket: https://theeventscalendar.atlassian.net/browse/TEC-4066

This is a patch so we don't need a change log. 

Artifact: https://www.loom.com/share/18e9fce6cf0142ab9d48ba946fa9e021

### CODE REVIEW

**Review by bordoni — CHANGES_REQUESTED** [review]
Needs Artifact.

**Comment by EugeneKyale** [comment]
I've added a video @bordoni

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 1]
```diff
@@ -176,6 +181,44 @@
...
...d: url("data:image/svg+xml;charset=utf-8,%3Csvg width='20' height='20' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M13.333 3.826c0 .065 0 .13-.02.174 0 .022-.02.065-.02.087a.9.9 0 0 1-.197.37L10.45 7.37a.797.797 0 0 1-.592.26.797.797 0 0 1-.593-.26c-.316-.348-.316-.935 0-1.305l1.225-1.348H6.3c-2.547 0-4.64 2.283-4.64 5.11 0 1.369.474 2.651 1.363 3.608.316.348.316.935 0 1.304A.797.797 0 0 1 2.43 15a.797.797 0 0 1-.593-.26C.652 13.434 0 11.695 0 9.847c0-3.826 2.825-6.935 6.301-6.935h4.208L9.284 1.565c-.316-.348-.316-.935 0-1.304.316-.348.85-.348 1.185 0l2.647 2.913a.952.952 0 0 1 .198.37c0 .021.02.065.02.086v.196zM20 10.152c0 3.826-2.825 6.935-6.301 6.935H9.49l1.225 1.348c.336.348.336.935 0 1.304a.797.797 0 0 1-.593.261.83.83 0 0 1-.592-.26l-2.627-2.936a.948.948 0 0 1-.198-.37c0-.021-.02-.064-.02-.086-.02-.065-.02-.109-.02-.174 0-.065 0-.13.02-.174 0-.022.02-.065.02-.087a.9.9 0 0 1 .198-.37L9.55 12.63c.316-.347.849-.347 1.185 0 .336.348.336.935 0 1.305L9.51 15.283h4.208c2.548 0 4.641-2.283 4.641-5.11 0-1.369-.474-2.651-1.362-3.608a.97.97 0 0 1 0-1.304c.316-.348.849-.348 1.185 0C19.348 6.543 20 8.283 20 10.152z' fill='var(--tec-color-icon-active)'/%3E%3C/svg%3E") 0/12px no-repeat;
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:195]: I don't think this works with the `var(--tec-color-icon-active)`, try another color and see, but if i remember correctly custom properties won't work in background url.

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 2]
```diff
@@ -247,6 +290,12 @@
 			font-size: 1.65rem;
 		}
 	}
+
+	/* Recurring event label */
+	.tribe-blocks-editor .tribe-events-single-event-recurrence-description span,
+	.tribe-blocks-editor .tribe-events-single-event-recurrence-description a {
+		font-size: 1.65rem;
+	}
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:298]: you can combine these

File: src/resources/postcss/tribe-events-single-blocks.pcss [hunk 3]
```diff
@@ -305,6 +354,12 @@
...
```
Comment (paulmskim) [inline src/resources/postcss/tribe-events-single-blocks.pcss:362]: you can combine these

**Review by paulmskim — CHANGES_REQUESTED** [review]
a few small things, but otherwise it's there!

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 1 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: Just confirmed it doesn't work with custom properties. We don't actually need the `fill` property here.

File: src/resources/postcss/tribe-events-single-blocks.pcss (hunk 2 above)
Comment (EugeneKyale) [reply src/resources/postcss/tribe-events-single-blocks.pcss]: I've nested these.

**Review by paulmskim — APPROVED** [review]

**Pull request merged**